console/Response.php

```diff
@@ -126,6 +136,29 @@ public function styles($styles = array()) {
 			'success' => "\033[0;32m",
 			'bold'    => "\033[1m",
 		);
+
+		if($this->noColor) {
+			$defaults = array(
+				'end'    => "",
+				'black'  => "",
+				'red'    => "",
+				'green'  => "",
+				'yellow' => "",
+				'blue'   => "",
+				'purple' => "",
+				'cyan'   => "",
+				'white'  => "",
+				'heading' => "",
+				'option'  => "",
+				'command' => "",
+				'error'   => "",
+				'success' => "",
+				'bold'    => "",
+			);
+
+			$styles = false;
+		}
+
 		if ($styles === false) {
```

Comment:
As far as I can see, you could remove the block above by changing this line to: 
`if ($styles === false || $this->noColor)` 

as a bonus round line 167-169 could also be removed by changing it to:

`if ($styles === false || $this->noColor || strtoupper(substr(PHP_OS, 0, 3)) === 'WIN')`